src/app/shared/search/search-export-csv/search-export-csv.component.html

```diff
@@ -1,7 +1,20 @@
+<ng-template #tipContent>
+    <div class="tooltip-content">
+        <p class="m-0">{{tooltipMsg | translate}}</p>
+    </div>
+</ng-template>
+
+<ng-template #tipContentWarning>
+    <div class="tooltip-content">
+        <p class="m-0">{{tooltipMsg | translate}}</p>
+        <p class="m-0 text-warning">{{exportLimitExceededMsg}}</p>
+    </div>
+</ng-template>
+
 <button *ngIf="shouldShowButton$ | async"
    class="export-button btn btn-dark btn-sm"
-   [ngbTooltip]="tooltipMsg | translate"
+   [ngbTooltip]="(shouldShowWarning$ | async) ? tipContentWarning : tipContent"
```

Comment:
We need to also find a way to display this warning in the `attr.aria-label` just a few lines below this.  That way this message is also shown to screen readers.